ocaml-lsp-server/src/state.ml

```diff
@@ -11,4 +11,5 @@ type t =
   ; scheduler : Scheduler.t
   ; detached : Fiber.Pool.t
   ; configuration : Configuration.t
+  ; trace : TraceValue.t
```

Comment:
The trace value is set, but not really used yet. It should maybe be used to update the `!level` function ?

Reply:
> The trace value is set, but not really used yet.

Yeah, I'm just not sure kind of messages need to be "traced". 

> It should maybe be used to update the !level function ?

That's one way to do it. Should trace receive non lsp related messages?